quad_control/setup.py

```diff
@@ -13,7 +13,9 @@
         'systems_functions',
         'quadrotor_tracking_controllers_hierarchical',
         'controllers_hierarchical',
-        'TrajectoryPlanner'
+        'TrajectoryPlanner',
+        'mission',
```

Comment:
This gives me an error: the name of the python package is `missions`, not `mission`.
No big deal, but I find it strange that it was working today. Ideas?